packages/plans-grid/src/plans-table/index.tsx

```diff
@@ -29,32 +31,85 @@ const PlansTable: React.FunctionComponent< Props > = ( {
 	onPickDomainClick,
 	currentDomain,
 } ) => {
+	const { __ } = useI18n();
+
 	const supportedPlans = useSelect( ( select ) => select( PLANS_STORE ).getSupportedPlans() );
 	const prices = useSelect( ( select ) => select( PLANS_STORE ).getPrices() );
-	const [ allPlansExpanded, setAllPlansExpanded ] = useState( false );
+
+	// Primary plan
+	// TODO: Feature picker determines recommended plan depending on the features that user choose.
+	const recommendedPlanSlug = null;
```

Comment:
This should be passed as a prop to plans-grid package and then passed to this component.

Reply:
Coming soon!